docs/taker.md

```diff
@@ -116,45 +118,175 @@ SUBCOMMANDS:
             Send to an external wallet address
 ```
 
-In order to do a coinswap, we first need to get some coins in our wallet. Let's generate a new address and send some coins to it.
+### Key Points About Command Arguments
+
+- The `-r` or `--ADDRESS:PORT` option specifies the Bitcoin Core RPC address and port. By default, this is set to `127.0.0.1:48332`.
+
+- The `-a` or `--USER:PASSWORD` option specifies the Bitcoin Core RPC authentication. By default, this is set to **`user:password`**.
+
+- #### If you're using the **default configuration**:
+
+  - You don't need to include these arguments.
+
+- #### If you're using a **custom configuration**:
+  - Pass your custom values using the `-r` and `-a` options, like this:
+
+```bash
+  $ ./taker -r 127.0.0.1:38332 -a myuser:mypass <SUBCOMMAND>
+```
+
+## For this tutorial, we'll assume a custom configuration with port 38332. Output examples will reflect this setup.
+
+---
+
+## Setting Up Your Wallet
+
+### Generate a New Address
+
+Before you can perform coinswaps, you need to fund your wallet. First, generate a new receiving address:
+
+```bash
+$ taker -r 127.0.0.1:38332 -a user:password get-new-address
```

Comment:
Once default changed to 38332, the args won't be needed. user:pssword are also default so no need to mention them.